commit e42b4a507efa19a90c63e7968c93c4f82d3bc805
Merge: 1e58e5e59148 e4e11cc0f81e
Author: Paolo Bonzini <pbonzini@redhat.com>
Date:   Fri Oct 19 15:24:24 2018 +0200

    Merge tag 'kvmarm-for-v4.20' of git://git.kernel.org/pub/scm/linux/kernel/git/kvmarm/kvmarm into HEAD
    
    KVM/arm updates for 4.20
    
    - Improved guest IPA space support (32 to 52 bits)
    - RAS event delivery for 32bit
    - PMU fixes
    - Guest entry hardening
    - Various cleanups

diff --cc include/uapi/linux/kvm.h
index cb6d44e1fe02,95aa73ca65dc..2b7a652c9fa4
--- a/include/uapi/linux/kvm.h
+++ b/include/uapi/linux/kvm.h
@@@ -960,11 -962,7 +969,12 @@@ struct kvm_ppc_resize_hpt 
  #define KVM_CAP_NESTED_STATE 157
  #define KVM_CAP_ARM_INJECT_SERROR_ESR 158
  #define KVM_CAP_MSR_PLATFORM_INFO 159
 -#define KVM_CAP_ARM_VM_IPA_SIZE 160 /* returns maximum IPA bits for a VM */
 +#define KVM_CAP_PPC_NESTED_HV 160
 +#define KVM_CAP_HYPERV_SEND_IPI 161
 +#define KVM_CAP_COALESCED_PIO 162
 +#define KVM_CAP_HYPERV_ENLIGHTENED_VMCS 163
 +#define KVM_CAP_EXCEPTION_PAYLOAD 164
++#define KVM_CAP_ARM_VM_IPA_SIZE 165
  
  #ifdef KVM_CAP_IRQ_ROUTING